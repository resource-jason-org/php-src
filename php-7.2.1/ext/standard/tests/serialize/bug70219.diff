002+ 
003+ Notice: unserialize(): Error at offset 55 of 56 bytes in /home/php-src/php-7.2.1/ext/standard/tests/serialize/bug70219.php on line 16
004+ bool(false)
002- array(2) {
003-   [0]=>
004-   object(obj)#%d (1) {
005-     ["data"]=>
006-     NULL
007-   }
008-   [1]=>
009-   &array(1) {
010-     ["data"]=>
011-     NULL
012-   }
013- }