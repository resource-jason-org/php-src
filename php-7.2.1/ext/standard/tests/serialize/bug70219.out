Warning: session_decode(): Failed to decode session object. Session has been destroyed in /home/php-src/php-7.2.1/ext/standard/tests/serialize/bug70219.php on line 9

Notice: unserialize(): Error at offset 55 of 56 bytes in /home/php-src/php-7.2.1/ext/standard/tests/serialize/bug70219.php on line 16
bool(false)